modules/samples-operator-overview.adoc

```diff
@@ -47,3 +47,20 @@ the following upon deletion:
 
 Upon deletion of the samples resource, the Samples Operator recreates the
 resource using the default configuration.
+
+[id="samples-operator-bootstrapped"]
+== Sample Operator's use of management state
+
+The Samples Operator is bootstrapped as `Managed` by default, which means that the Operator is actively managing its resources and keeping the component active in order to pull sample imagestreams and images from the registry.
+
+Certain circumstances will result in the Samples Operator bootstrapping itself as `Removed` including:
+
+* If the Samples Operator cannot reach link:registry.redhat.io[registry.redhat.io] after three minutes on initial start-up after installation.
+* If the Samples Operator detects it is on an IPv6 network.
+* If global proxy is configured.
```

Comment:
i probably was not clear ... if global proxy is detected, we do *NOT* bootstrap as removed 

the checks are in the following order:
- if ipv6, boot as removed
- else if global proxy cfged, boot as managed 
- else if cannot connnect to registry.redhat.io after 3 minute on startup after initial install, boot as removed 

and by "initial install" I mean a clean install ... these checks are not done following an upgrade